README.md

```diff
@@ -83,6 +83,19 @@ See
 [the gatsby-plugin-netlify docs](https://github.com/netlify/gatsby-plugin-netlify/)
 for more information, including optional plugin configuration.
 
+### Disabling Netlify functions
+
+In order to support Gatsby Functions and DSG and SSR render modes, this plugin
+generates three Netlify Functions called `__api`, `__ssr` and `__dsg`. If you
+are not using any of these modes, then you can disable the creation of these
+functions. If you are using the latest version of `gatsby-plugin-netlify` then
+this will be handled automatically, by the plugin generating the file
+`.cache/.nf-skip-gatsby-functions` if it detects than there are no DSG or SSR
```

Comment:
```suggestion
`.cache/.nf-skip-gatsby-functions` and if it detects it, then the plugin knows there are no DSG or SSR
```
Sentence wasn't making much sense, maybe something like this?

Reply:
I'll re-word it, because that's not what it means.

Reply:
@tiffanosaurus How about now?